### DISCUSSION TITLE
Problem with hotkey B + Select

### DISCUSSION THREAD

**Discussion by MarcoEstevez** [eduardofilo/RG350_adam_image#238] [RetroArch topics]
In Retroarch, B + Select does reset the game, but Final Fantasy I, uses B + Select to show the map, how to disable this hotkey?

**Comment by neilswann80** [comment]
With the game running...
- Press `POWER` to bring up the RetroArch Quick Menu
- Press `B` to go to the Main Menu
- Select `Restart RetroArch`
- Go to `Settings>Input>Hotkeys`
- Scroll down to `Reset Game` and press `Y` to clear
- Press `B` until you're back to the Main Menu
- Select `Configuration file`
- Select `Save Current Configuration`
- Press `B` then `Quit RetroArch`

Try the game again, hopefully the RetroArch hotkey is now disabled.

**Reply by eduardofilo** [reply]
Another option is to disable that hotkey only on the platform of that game (SNES?) or even only on that game by doing an override at the content or game level. That's if you prefer to keep the reset hotkey in all other situations.

**Reply by MarcoEstevez** [reply]
It worked great thanks, ill try override it at core level.
